docs/docs/API-Reference/api-reference-api-examples.md

```diff
@@ -321,50 +415,85 @@ curl -X POST \
+// ... Additional events as the flow executes ...
 
 {"event": "end", "data": {}}
 ```
 
    </TabItem>
 </Tabs>
 
-This output is abbreviated, but the order of events illustrates how Langflow runs components.
-
-1. Langflow first sorts the vertices by dependencies (edges) in the `vertices_sorted` event:
-```
-ChatInput → Prompt → OpenAIModel → ChatOutput
+The events endpoint accepts an optional `stream` query parameter which defaults to `true`.
+To disable streaming and get all events at once, set `stream` to `false`.
+```curl
+curl -X GET \
+  "$LANGFLOW_URL/api/v1/build/123e4567-e89b-12d3-a456-426614174000/events?stream=false" \
+  -H "accept: application/json"
 ```
-2. The Chat Input component receives user input in the `add_message` event.
-3. The Prompt component is built and executed with the received input in the `end_vertex` event.
-4. The Open AI model's responses stream as `token` events.
-The `token` event represents individual pieces of text as they're generated by an LLM.
-5. The clean `end` event tells you the flow executed with no errors.
-If your flow executes with errors, the `error` event handler prints the errors to the playground.
+
+### Build endpoint headers and parameters
+
+**Headers**
+| Header | Info | Example |
+|--------|------|---------|
+| Content-Type | Required. Specifies JSON format | "application/json" |
+| accept | Required. Specifies response format | "application/json" |
+| x-api-key | Optional. Required only if authentication is enabled | "sk-..." |
+
+The `/build/{flow_id}/flow` endpoint accepts the following parameters in its request body:
+
+**Parameters**
+| Parameter | Type | Description |
+|-----------|------|-------------|
+| inputs | object | Optional. Input values for flow components. |
+| data | object | Optional. Flow data to override stored configuration. |
+| files | array[string] | Optional. List of file paths to us. |
```

Comment:
..."file paths to us" - who is "us" referring to? Langflow?

Reply:
```suggestion
| files | array[string] | Optional. List of file paths to use. |
```